modules/local/crisprseq_plotter.nf

```diff
@@ -0,0 +1,42 @@
+process CRISPRSEQ_PLOTTER {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda 'r-ggplot2=3.4.3 bioconductor-shortread=1.58.0 r-ggpubr=0.6.0 r-ggmsa=1.0.2 r-seqmagick=0.1.6 r-tidyr=1.3.0 r-ggseqlogo=0.1 r-cowplot=1.1.1 r-seqinr=4.2_30 r-optparse=1.7.3 r-dplyr=1.1.2 r-plyr=1.8.8 r-stringr=1.5.0 r-plotly=4.10.2'
+    // TODO: change containers
```

Comment:
Reminder for this TODO

Reply:
Removing the comment as they were changed by https://github.com/nf-core/crisprseq/pull/74/commits/4acfc21892511cda60db6a8a5c358dc10f317465
thanks!
```suggestion
```